pyramid/scripts/pserve.py

```diff
@@ -334,6 +341,15 @@ def serve():
                     msg = ''
                 self.out('Exiting%s (-v to see traceback)' % msg)
 
+        if self.options.webbrowser_open:
+            def open_browser():
+                context = loadcontext(SERVER, app_spec, name=app_name, relative_to=base,
+                        global_conf=vars)
+                url = 'http://{host}:{port}/'.format(**context.config())
+                time.sleep(1)
+                webbrowser.open(url)
+            threading.Thread(target=open_browser).start()
```

Comment:
can we mark the thread as detached so that it doesn't block the process from terminating?